gradle-plugin/it/src/it/debug-mode/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'org.eclipse.jkube.kubernetes' version "${jKubeVersion}"
+    id 'org.eclipse.jkube.openshift' version "${jKubeVersion}"
+    id 'java'
+}
+
+group = 'org.eclipse.jkube.integration.tests.gradle'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+    mavenCentral()
+}
+
+kubernetes {
+    offline = true
+    images {
+        image {
+            name = 'repository/debug-mode:latest'
+            build {
+                from = 'alpine'
+                assembly {
+                    targetDir = '/deployments'
+                }
+                entryPoint {
+                    shell = "java -jar /deployments/${project.name}-${project.version}.jar"
+                }
+            }
+        }
+    }
+}
+
+openshift {
+    offline = true
+    images {
+        image {
+            name = 'repository/debug-mode:latest'
+            build {
+                from = 'alpine'
+                assembly {
+                    targetDir = '/deployments'
+                }
+                entryPoint {
+                    shell = "java -jar /deployments/${project.name}-${project.version}.jar"
+                }
+            }
+        }
+    }
+}
```

Comment:
Any reason why we are using a custom image for this test?

Could we use something that, at least, relies on our base image that **does** have support for debug mode?

Reply:
When using custom image, we don't need to provide extra files in `src/` directory

Reply:
Are you asking to change `from` to some jkube base image? Bdw, this dummy image configuration is provided just to ensure resources are generated.